src/ExpressionEvaluator/Core/Source/ExpressionCompiler/PDB/PdbHelpers.cs

```diff
@@ -82,176 +81,29 @@ public static ConstantValue GetSymConstantValue(ITypeSymbol type, object symValu
                 type = ((INamedTypeSymbol)type).EnumUnderlyingType;
             }
 
-            short shortValue;
-            switch (type.SpecialType)
+            return (type.SpecialType, symValue) switch
             {
```

Comment:
❤️❤️❤️❤️❤️❤️

Reply:
Great to know!  For some reason i had internalized `when` clauses and the like caused linearization.  I'll def be using this more now that i know it's safe!